### ISSUE TITLE
[1.18.1 Fabric Bug] Patchouli Empty Content error warning?

### ISSUE THREAD

**Issue by itsdinkd** [AzureDoom/MCDoom#38]
Using 4.18, this is appearing in latest.log, though, I personally haven't checked the guide as I just randomly came across this while reading my logs for any errors. 

Placing here in case this wasn't known

```java
[20:45:12] [Render thread/ERROR]: Error compiling book doom:doom, using empty contents
java.lang.RuntimeException: Error building entry doom:demons/pinky
	at Not Enough Crashes deobfuscated stack trace.(1.18.1+build.2) ~[?:?]
	at vazkii.patchouli.client.book.BookContentsBuilder.lambda$build$2(BookContentsBuilder.java:98) ~[Patchouli-1.18.1-65-FABRIC.jar:?]
	at java.util.HashMap$Values.forEach(HashMap.java:1065) ~[?:?]
	at vazkii.patchouli.client.book.BookContentsBuilder.build(BookContentsBuilder.java:94) ~[Patchouli-1.18.1-65-FABRIC.jar:?]
	at vazkii.patchouli.common.book.Book.reloadContents(Book.java:197) ~[Patchouli-1.18.1-65-FABRIC.jar:?]
	at vazkii.patchouli.common.book.BookRegistry.reloadContents(BookRegistry.java:128) ~[Patchouli-1.18.1-65-FABRIC.jar:?]
	at vazkii.patchouli.client.book.ClientBookRegistry.reload(ClientBookRegistry.java:63) ~[Patchouli-1.18.1-65-FABRIC.jar:?]
	at vazkii.patchouli.client.base.ClientAdvancements.onClientPacket(ClientAdvancements.java:34) ~[Patchouli-1.18.1-65-FABRIC.jar:?]
	at net.minecraft.client.network.ClientAdvancementManager.handler$igi000$patchouli_onSync(ClientAdvancementManager:1517) ~[?:?]
	at net.minecraft.client.network.ClientAdvancementManager.onAdvancements(ClientAdvancementManager:57) ~[?:?]
	at net.minecraft.client.network.ClientPlayNetworkHandler.onAdvancements(ClientPlayNetworkHandler:1208) ~[?:?]
	at net.minecraft.network.packet.s2c.play.AdvancementUpdateS2CPacket.apply(AdvancementUpdateS2CPacket:51) ~[?:?]
	at net.minecraft.network.packet.s2c.play.AdvancementUpdateS2CPacket.apply(AdvancementUpdateS2CPacket:16) ~[?:?]
	at net.minecraft.network.NetworkThreadUtils.method_11072(NetworkThreadUtils:21) ~[?:?]
	at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor:151) ~[?:?]
	at net.minecraft.util.thread.ReentrantThreadExecutor.executeTask(ReentrantThreadExecutor:23) ~[?:?]
	at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor:125) ~[?:?]
	at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor:110) ~[?:?]
	at net.minecraft.client.MinecraftClient.render(MinecraftClient:1080) ~[?:?]
	at net.minecraft.client.MinecraftClient.run(MinecraftClient:733) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:238) [client-intermediary.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) [fabric-loader-0.13.1.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.1.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.13.1.jar:?]
Caused by: java.lang.RuntimeException: Error while loading entry doom:demons/pinky page 2
	at vazkii.patchouli.client.book.BookEntry.build(BookEntry.java:224) ~[Patchouli-1.18.1-65-FABRIC.jar:?]
	at vazkii.patchouli.client.book.BookContentsBuilder.lambda$build$2(BookContentsBuilder.java:96) ~[Patchouli-1.18.1-65-FABRIC.jar:?]
	... 26 more
Caused by: java.lang.RuntimeException: Unknown entity id: doom:pinky2016
	at vazkii.patchouli.common.util.EntityUtil.loadEntity(EntityUtil.java:48) ~[Patchouli-1.18.1-65-FABRIC.jar:?]
	at vazkii.patchouli.client.book.page.PageEntity.build(PageEntity.java:46) ~[Patchouli-1.18.1-65-FABRIC.jar:?]
	at vazkii.patchouli.client.book.BookEntry.build(BookEntry.java:221) ~[Patchouli-1.18.1-65-FABRIC.jar:?]
	at vazkii.patchouli.client.book.BookContentsBuilder.lambda$build$2(BookContentsBuilder.java:96) ~[Patchouli-1.18.1-65-FABRIC.jar:?]
	... 26 more
```

**Comment by AzureDoom** [comment]
oh fuck, good catch. This is due to the mob squish I did. Update will be out soon!

**Comment by AzureDoom** [comment]
Fixed in 4.1.20
